platform/broadcom/sonic-platform-modules-cel/silverstone/modules/switchboard.c

```diff
@@ -25,7 +25,7 @@
  */
 
 #ifndef TEST_MODE
-#define MOD_VERSION "1.2.0"
+#define MOD_VERSION "1.3.0"
```

Comment:
This should be rebased since the other PR #90 merged

Reply:
sure, updated